hack/install-deps.sh

```diff
@@ -29,25 +29,31 @@ function cmd() {
 # so we can amend our path to look into the local node_modules for the
 # correct binaries.
 repo_root="$( dirname "${BASH_SOURCE}" )/.."
-export PATH="${PATH}:${repo_root}/node_modules/bower/bin:${repo_root}/node_modules/grunt-cli/bin"
+export PATH="${PATH}:${repo_root}/node_modules/grunt-cli/bin"
 
-# Install bower if needed
-if ! which bower > /dev/null 2>&1 ; then
-  cmd "npm install bower"
+# Install yarn if needed
+if ! which yarn > /dev/null 2>&1 ; then
+  cmd "curl -o- -L https://yarnpkg.com/install.sh | bash"
```

Comment:
This seems to fail in Jenkins. Maybe `npm install --global yarn` ?

Reply:
That way it will also install just for this repo rather than installing it globally.

Reply:
I tried that but there is a write permissions error when trying to install yarn globally. We could try just a local `npm install yarn` and see if that works.

Reply:
Yeah I ninja edited my comment because I didn't mean `--global`. I don't think we'd want `--global` anyway.